plugins/common/testing/api.ts

```diff
@@ -0,0 +1,8 @@
+import { UnitTestResult } from "./type";
+
+export interface TestingApi {
```

Comment:
I wouldn't right now put this is in common api cause this is not core module / feature. can we wait a bit?

Reply:
For me this is the most urgent one, as we need to be able to provide a unified interface for the workshop plugin.

Reply:
well honestly, less urgent than fileSystem, editor, as you said we need it only for the workshop plugin and improving workshop plugin will not come in the next days (weeks)
That's not a core thing like editor, filesystem, ...

Reply:
Ok if you want I remove it.